website/src/util/hasOnlyUndefinedValues.ts

```diff
@@ -0,0 +1,19 @@
+export function hasOnlyUndefinedValues(obj: any): boolean {
+    if (typeof obj !== 'object' || obj === null) {
+        return false;
+    }
```

Comment:
```suggestion
export function hasOnlyUndefinedValues(obj: object): boolean {
    if (obj === null) {
        return true;
    }
```

Reply:
Maybe this is even better:
```suggestion
export function hasOnlyUndefinedValues(obj: {}): boolean {
```

eslint will complain:
>The `{}` ("empty object") type allows any non-nullish value, including literals like `0` and `""`.
>- If that's what you want, disable this lint rule with an inline comment or configure the 'allowObjectTypes' rule option."

But I think in this case, it's actually what we want? It doesn't allow `null` and we don't care about values or keys, because we check them anyway.

Reply:
I took the version with `obj: object`, since `0` and `""` would be possible in the `{}` solution.